Composer/packages/client/src/pages/design/ManifestEditor.tsx

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import formatMessage from 'format-message';
+import ErrorBoundary from 'react-error-boundary';
+import React from 'react';
+import { LoadingTimeout } from '@bfc/adaptive-form/lib/components/LoadingTimeout';
+import { FieldLabel } from '@bfc/adaptive-form/lib/components/FieldLabel';
+import ErrorInfo from '@bfc/adaptive-form/lib/components/ErrorInfo';
+import { FontSizes } from '@uifabric/fluent-theme';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { FontWeights } from '@uifabric/styling';
+import { DetailsList, DetailsListLayoutMode, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
+import get from 'lodash/get';
+
+import { SkillInfo } from '../../recoilModel';
+
+import { formEditor } from './styles';
+
+const styles = {
+  errorLoading: css`
+    padding: 18px;
+  `,
+
+  banner: css`
+    border-bottom: 1px solid #c8c6c4;
+    padding: 0 18px;
+    margin-bottom: 0px;
+  `,
+
+  title: css`
+    font-size: ${FontSizes.size20};
+    font-weight: ${FontWeights.semibold};
+    margin: '5px 0 7px -9px';
```

Comment:
same issue with negative margins here. Seems like the parent container css needs to be adjusted?